scripts/release.sh

```diff
@@ -1,101 +0,0 @@
-#!/bin/bash
```

Comment:
why this script was deleted?

Reply:
It is unused for a long time now. In the very early days (before IOV-Core was on npm), we used this script together with webpack to bundle the library for internal testing. I saw it again when I deleted the also unused "pack-node" script.